speedcord/shard.py

```diff
@@ -12,6 +12,12 @@
 
 class DefaultShard:
     def __init__(self, shard_id, client, loop: AbstractEventLoop):
+        """
+        Handles all Discord Shard related events. For more information on what sharding is and how it works: https://discord.com/developers/docs/topics/gateway#sharding. 
```

Comment:
Discord.dev link? Also maybe make it clear that this is a discord shard not the handler for shard events?